{"name": "test_send_http[model8]", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range", "trace": "self = <DiaoApp.case.test_httpRun.Test_case object at 0x000001AEEE834F40>\nmodel = <DiaoApp.model.http_model.http_model object at 0x000001AEEE0AB880>\nkwargs = {}, case_num = 9, last_value_list = ['id', 'maxPrice'], i = 1\nvalue = 'maxPrice', data_value = 188.0, up_key_list = ['auctionid']\nup_key = 'auctionid'\n\n    @allure.title(\"\")  # 断言完成，更新标题\n    @pytest.mark.parametrize(\"model\", models)\n    def test_send_http(self, model, **kwargs):\n        '''\n        :model:传入的数据对象\n        :param kwargs:\n        :return:\n        '''\n        # 记录当前用例顺序\n        case_num = self.models.index(model) + 1\n    \n        allure.step(f\"请求URL: {model.url}\")\n        allure.step(f\"请求方式: {model.method}\")\n        allure.step(f\"请求头: {model.headers}\")\n        allure.step(f\"请求数据: {model.data}\")\n        allure.step(f\"依赖数据: {model.dependence_case}\")\n        allure.step(f\"预期数据: {model.assert_value}\")\n    \n        model.headers = comn().get_headers()\n        if model.is_need:  # 判断是否需要抽取的值\n            if self.contentData:\n                print(self.contentData)\n                # 获取登录token并传入header\n                model.headers = comn().get_headers()\n                model.headers[config['token']] = self.contentData[config['res_code']]\n    \n                # 通过切片获取需要取的值，因为code在第一个所以下标从1往后取\n                last_value_list = model.last_value.split(\",\")[1:]\n                for i in range(len(last_value_list)):\n                    value = last_value_list[i]\n                    data_value = self.contentData[value]\n    \n                    # 修改data中需要更新的字段\n                    # 注：case中update_key中的值必须与对应last_value中code后的值以及顺序一致\n                    if model.update_key is not None:\n                        up_key_list = model.update_key.split(\",\")\n>                       up_key = up_key_list[i]\nE                       IndexError: list index out of range\n\ncase\\test_httpRun.py:74: IndexError"}, "description": "\n        :model:传入的数据对象\n        :param kwargs:\n        :return:\n        ", "parameters": [{"name": "model", "value": "<DiaoApp.model.http_model.http_model object at 0x000001AEEE0AB880>"}], "start": 1660812432762, "stop": 1660812432762, "uuid": "22568947-a2df-4d43-85d2-7d0bac9ce4a2", "historyId": "a2f8de0a636c83387662b5a549a93c8e", "testCaseId": "ed31b99d3b3aa454dea09452c91e8949", "fullName": "case.test_httpRun.Test_case#test_send_http", "labels": [{"name": "parentSuite", "value": "case"}, {"name": "suite", "value": "test_httpRun"}, {"name": "subSuite", "value": "Test_case"}, {"name": "host", "value": "OS2AQ70MWVFP3P9"}, {"name": "thread", "value": "21288-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.test_httpRun"}]}